add config.json sample to the README.md example (#4) (#5)
ossf-cve-benchmark/ossf-cve-benchmark: docs/configuration.md

## Configuring drivers for analysis tools

Configuration files are also used to configure drivers for analysis
tools.  For an example of configuring a driver for an analysis tool,
see [Configuring the driver for
`eslint`](eslint-example.md#configuring-the-driver-for-eslint), or
browse the README.md files of each driver in
[contrib/tools](../contrib/tools).

## Sample configuration files

This section contains various `config.json` examples.

For more information about the allowed contents of a configuration file, 
see the JSON schema for the [Config](json-schema/ts-defs-definitions-config.md) type.

### Minimal configuration file

The configuration file does not need to specify anything, but if
`config.json` is present, it should at least be a valid JSON object.

```json
{}
```

### Configuration file with one configured driver

> As mentioned above, the `.tools` property of the configuration file is
> used to configure drivers for analysis tools. The sample below makes
> it possible to use `bin/cli run --tool eslint-default` to benchmark
> ESLint.

Reviewer: Reworded to be more active.

```suggestion
As mentioned above, the `.tools` property of the configuration file is
used to configure drivers for analysis tools. The example below configures 
a driver for ESLint. You can then benchmark ESLint by running 
`bin/cli run --tool eslint-default`.
```

```json
{
  "tools": {
    "eslint-default": {
      "bin": "node",
      "args": [
        "/home/user-name/ossf-cve-benchmark/build/ts/contrib/tools/eslint/src/eslint.js",
      ],
      "options": {
        "eslintDir": "/home/user-name/analysis-tools/eslint-2020-12-08"
      }
    }
  }
}
```

### Configuration file with two configured drivers

> It is rarely useful to only have one configured driver for
> benchmarking. The sample below makes it possible to use `bin/cli run
> --tool eslint-default --tool eslint-2019` to benchmark two different
> versions of ESLint.

Reviewer: As above.

```suggestion
It is rarely useful to only have one configured driver for
benchmarking. The sample below configures drivers for two different
versions of ESLint. You can benchmark both versions to compare their performance
by running `bin/cli run --tool eslint-default --tool eslint-2019`.
```
